WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderFormGiftCardRow.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+/// Gift card entered by the user in the order form.
+struct OrderFormGiftCardRow: View {
+    let code: String
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            HStack(spacing: Constants.horizontalSpacing) {
+                Text(Localization.giftCard)
+                    .foregroundColor(.init(uiColor: .accent))
+                    .bodyStyle()
+                Image(systemName: "pencil")
```

Comment:
I see there are some pencil icons in `UIImage+Woo` should we be using those for consistency?

Reply:
The pencil looks a bit thick, but I agree to keep the consistency (updated in https://github.com/woocommerce/woocommerce-ios/pull/10791/commits/8940721c06b8f4522fba91ee991ad9301f7cb29f):

<img src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/652152e3-a1f6-41f6-a8c4-53eb5a10810f" width="300" />